### DISCUSSION TITLE
Possiblity to always insert future date

### DISCUSSION THREAD

**Discussion by Gnopps** [argenos/nldates-obsidian#70] [Ideas]
Thanks for developing this plugin!

One thing I found is that sometimes past dates are inserted, whereas my use case are to always insert future dates. For example, today is Friday. If I today parse "fri" then last week's Friday will be inserted, but I'd like next weeks Friday. Same goes for other days of the week. I know I can type "next fri" but that still involves more typing.

So my wish would be for a setting to change if default is to insert past or future date.

**Comment by argenos** [comment]
I'm honestly not sure that I will add a setting for this. It adds development overhead for edge cases and I think using "next" when you do mean "next" is a good-enough solution.

**Comment by charlie17** [comment]
In a related but strange finding, sitting here today (Tuesday evening), if I enter `@wed` it inserts tomorrow's date - good. If I enter `@thu` it inserts the day after tomorrow's date - good.

But if I enter `@fri` it inserts LAST Friday's date. Same with `@sat` or `@sun`.

Seems like it should certainly should insert upcoming/future dates for "this week"?

Thanks

**Comment by jasonhaslow-ui** [comment]
I have a hard time with this as well.  For instance: 
@today => 2023-08-10 (which is a Thursday)
@last Tuesday => 2023-07-25

I can see arguments for:

- 2023-08-08, this is the most recent Tuesday in the past so accurate from a pure language perspective, but I don't think is what most people mean when `last Tuesday` in the context of a week.   
- 2023-08-01, last week's Tuesday, not strictly true from language perspective, but I believe is what most people mean when the say `last Tuesday`.  

I don't understand why it would go all the way back to 2023-07-25.
